### ISSUE TITLE
--track parameter not working appropriately in SaaS

### ISSUE THREAD

**Issue by 111andre111** [elastic/ecctl#228]
## Expected Behavior

Track filter should show what happens.

## Current Behavior

In #221 I got the hint about the `--track` parameter.
Unfortunately it doesn't work out for the moment with SaaS.
`ecctl deployment create --file ./deployment.json --name=my-new-deployment1 --version=7.4.1 --track`
I get the error message:
```
{
  ... credentials JSON is displayed
}
Cluster [a9.................][Apm]: caught error: "&{[]} (*models.BasicFailedReply) is not supported by the TextConsumer, can be resolved by supporting TextUnmarshaler interface" (Total plan duration: 1m10.852099986s)
Cluster [1b.................][Elasticsearch]: caught error: "&{[]} (*models.BasicFailedReply) is not supported by the TextConsumer, can be resolved by supporting TextUnmarshaler interface" (Total plan duration: 1m10.978045332s)
Cluster [c9.................][Kibana]: caught error: "&{[]} (*models.BasicFailedReply) is not supported by the TextConsumer, can be resolved by supporting TextUnmarshaler interface" (Total plan duration: 1m11.130569337s)
3 errors occurred:
	* cluster [a9.................][apm] &{[]} (*models.BasicFailedReply) is not supported by the TextConsumer, can be resolved by supporting TextUnmarshaler interface
	* cluster [1b.................][elasticsearch] &{[]} (*models.BasicFailedReply) is not supported by the TextConsumer, can be resolved by supporting TextUnmarshaler interface
	* cluster [c9.................][kibana] &{[]} (*models.BasicFailedReply) is not supported by the TextConsumer, can be resolved by supporting TextUnmarshaler interface

```

Reference:
https://github.com/elastic/ecctl/issues/221

**Comment by marclop** [comment]
This is known, #226 will fix it.

**Comment by marclop** [comment]
This has been solved by #226 and available in master, it'll be published in the next release
